README.md

```diff
@@ -19,6 +19,17 @@ For example, add the following to your config.php after the `$CFG->wwwroot` line
    define('TEST_MOD_BIGBLUEBUTTONBN_MOCK_SERVER', "http://localhost:8001/hash" . sha1($CFG->wwwroot));
    ```
 
+### Moodle Docker
+To run automated tests in conjunction with the [Moodle Docker](https://github.com/moodlehq/moodle-docker) testing environment you
+should start the mock server as above, then determine the gateway address of the webserver container, e.g.:
+   ```
+   docker inspect -f "{{ .NetworkSettings.Networks.docker_default.Gateway }}" docker-webserver-1
```

Comment:
Err... what is `docker-webserver-1`?

Typically if you're using this in the moodle-docker container, the simpler option is to add a new yml file which configures this host, and then use the hostname.

Any docker-compose configuration sets up DNS.

Reply:
> Err... what is docker-webserver-1?

It's the name of the running container created when you start moodle-docker